src/bin/cli.ts

```diff
@@ -1,5 +1,5 @@
 import * as path from 'path'
-import * as fs from 'fs'
+import * as fs from 'mz/fs'
```

Comment:
Now that we're Node 8 only, we could use `util.promisify` to get rid of one dependency.

Reply:
yes!